tests/log-messages.feature

```diff
@@ -0,0 +1,32 @@
+# This file is part of REANA.
+# Copyright (C) 2023 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+Feature: Log messages
+
+    As a researcher,
+    I want to be able to see the log messages of my workflow execution,
+    So that I can verify that the workflow ran correctly.
+
+    Scenario: The workflow start has produced the expected messages
```

Comment:
```
When the workflow finishes 
Then the status should be "finished"
```

This sounds a bit like a tautology, and it may detract users from the focus of the scenario. I would leave this test out?

```
When the workflow finishes
Then the engine logs should contain ...
```

I think this reads much better...

P.S. This applies to all the other scenarios here and in the other feature files.